# RobotArmCommand.msg
int32 servo1_id
float32 servo1_pos
float32 servo1_time

int32 servo2_id
float32 servo2_pos
float32 servo2_time

int32 servo3_id
float32 servo3_pos
float32 servo3_time

int32 servo4_id
float32 servo4_pos
float32 servo4_time

int32 servo5_id
float32 servo5_pos
float32 servo5_time

int32 servo6_id
float32 servo6_pos
float32 servo6_time


